scripts/postgres_verification_commands.sh

```
@@ -0,0 +1,141 @@
+# Check if DATABASE_URL is set
+if [ -z "$DATABASE_URL" ]; then
+    echo "❌ DATABASE_URL not set. Please export it first:"
+    echo "export DATABASE_URL='postgresql://user:pass@localhost/dealgenie'"
+    exit 1
+fi
+
+echo "Database URL: $DATABASE_URL"
+echo ""
+
+# Helper variable for cleaner psql commands
+PSQL="psql \"$DATABASE_URL\""
```

Shellcheck (0.10.0)

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)